concent_api/core/validation.py

```diff
@@ -334,3 +335,25 @@ def validate_compute_task_def(compute_task_def: message.tasks.ComputeTaskDef) ->
             )
 
     validate_scene_file(extra_data['scene_file'])
+
+
+def validate_if_given_object_is_expected_enum_instance(
+    custom_object: enum.Enum,
+    base_enum: type,
+    allow_none_as_instance: bool = False,
+) -> None:
+
+    if allow_none_as_instance is True and custom_object is None:
+        return
+
+    if allow_none_as_instance is False and custom_object is None:
```

Comment:
Please replace `allow_none_as_instance is False` with simply `not allow_none_as_instance`.

We only do `if x is Y:` (e.g. `if custom_object is None:`) when `x` is not a boolean expression. Here it is so you don't have to add `is True` or `is False` to convert it.

Reply:
I personally don't like `not` in this context. For me `is False` is more readable as `not`, but it is only my personal preference.. It is changed :)